client/src/helpers/functions.js

```diff
@@ -0,0 +1,5 @@
+export const isArabic = (text) => {
+  const pattern = /[\u0600-\u06FF]/;
+  const result = pattern.test(text);
```

Comment:
```suggestion
 return pattern.test(text);
```

this can be simplified as we are not using the result value.